src/microbe_stage/editor/MicrobeEditor.tscn

```diff
@@ -3815,7 +3816,6 @@ margin_bottom = 311.0
 mouse_filter = 1
 min_value = -10.0
 max_value = 10.0
-step = 1.0
```

Comment:
I assume this doesn't break the rigidity slider?

Reply:
works fine